django_project/certification/models/course.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+"""
+Course model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.text import slugify
+from core.settings.contrib import STOP_WORDS
+from unidecode import unidecode
+from attendee import Attendee, Certificate
+
+
+class Course(models.Model):
+    """Course model."""
```

Comment:
rather start with the fields and then continue with the functions than interlieving these.